Gemfile

```diff
@@ -87,6 +89,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'rails-controller-testing'
   gem 'rspec-rails', '~> 4.0.0'
+  gem 'timecop'
```

Comment:
TIL! This is super helpful, since this has historically been a one-person operation I've let the tests slide but this will be a positive addition I think.

Reply:
It's a really handy gem, I use it in most of my projects